core/status_download.js

```diff
@@ -0,0 +1,124 @@
+//
+// When a photo/video status is being viewed, create a button to download it that appears, and then handle downloading it
+//
+
+function determineIfNodeIsStatus(node) {
+  // a correct status photo/img has the following parent classlists: 5, 0, 1, 5
+  // it's not a perfect solution
```

Comment:
For alternative ways to detect status (a few ideas, tell me which ones you think will work better):

- 8 parents up from the image/video there's a sibling element that creates the blurred background that's not present for other photos?
- 9 parents up from the image/video there's div elements that contain 9 child elements down the text "Type a reply..."
- In the places I have checked so far, all images have some form on inline styling that is not just "visibilty:visible" so this could be detected, but it seems like this solution will be more unreliable?

Can you think of any others?

Reply:
I am not in front of the status DOM right now, but I would try to find indicative words in the HTML like "status". I you don't see any, maybe you can detect the "Type a reply" textbox.

Reply:
I found this that is fairly close to the image in the DOM, so I will write some code to check for the SVG button that allows you to pause/play a status 
![image](https://github.com/tomer8007/whatsapp-web-incognito/assets/74068072/8d001b47-e187-46a2-8fbb-b0958a0af5ec)

Reply:
Yes it does, because it is used to prevent it from automatically moving on the slideshow of pictures. However, the other one next to it does behave differently because it controls audio that is obviously irrelevant to a photo.
![image](https://github.com/tomer8007/whatsapp-web-incognito/assets/74068072/ee2a6ac6-62ec-4579-9572-0264ab217d88)

Reply:
OK